src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -31,11 +33,26 @@ export class SearchResults extends Component {
     updatePagination: PropTypes.func.isRequired,
   };
 
-  componentDidUpdate(previousProps) {
-    const justRefreshed = previousProps.fetching && !this.props.fetching;
+  constructor(props) {
+    super(props);
 
-    if (justRefreshed) {
+    this.state = { howToSort: '', sortedResults: [] };
+    this.sortOptions = ['Last Updated (Newest)', 'Last Updated (Oldest)'];
+  }
+
+  componentDidMount() {
+    const { props, updateSortedResultsState } = this;
+    if (props.results && props.results.length > 0) updateSortedResultsState();
+  }
+
+  componentDidUpdate(previousProps, previousState) {
```

Comment:
So this makes the sorting a side effect of:

1) refetching the data
2) `howToSort` changes

Could we instead not use `componentDidUpdate` and instead just sort the data right when `howToSort` changes and on receiving the response data when fetching?